.github/workflows/osv-scanner-reusable.yml

```diff
@@ -56,6 +56,10 @@ on:
         description: "Whether to check out submodules or not. Passed on to `submodules` argument of `actions/checkout`"
         type: boolean
         default: false
+      ref:
+        description: "Whether to checkout a custom branch/tag/commit for scanning other than the default branch"
+        type: string
+        default: ""
```

Comment:
Tested passing empty string to `action/checkout,ref:` will result in the default branch being checked out,